Networking/Sources/Networking/Peer.swift

```diff
@@ -417,6 +438,12 @@ private struct PeerEventHandler<Handler: StreamHandler>: QuicEventHandler {
             )
             return
         }
+        // Check if the connection is already reconnected
+        impl.reconnectStates.write { reconnectStates in
+            if reconnectStates[conn.remoteAddress] != nil {
+                reconnectStates[conn.remoteAddress] = nil
+            }
```

Comment:
```suggestion
            reconnectStates[conn.remoteAddress] = nil
```
it is the same, the check is pointless